package.json

```diff
@@ -18,7 +18,8 @@
     "@stablelib/random": "^1.0.0",
     "base58-universal": "^1.0.0",
     "crypto-ld": "^4.0.2",
-    "esm": "^3.2.25"
+    "esm": "^3.2.25",
+    "node-forge": "^0.10.0"
```

Comment:
@davidlehn how do I ensure that node-forge is not loaded in the browser? An alias correct?

Reply:
used an external and it works